platalea/audio/features.py

```diff
@@ -46,8 +48,8 @@ def raw_frames(data, frame_shift, window_size):
     frames = []
     energy = []
 
-    for f in range (0, nframes):
-        frame = data[f * frame_shift : f * frame_shift + window_size]
+    for f in range(0, nframes):
+        frame = data[(f * frame_shift):(f * frame_shift + window_size)]
```

Comment:
Yeah it was interesting. The linter complained about the space in front of the colon. Not about the one behind the colon, strangely. However, when you write it without the parentheses, it becomes hard to read, because the colon is hard to notice, so it looks more like a single index with a long, complex calculation, than a range-index. I think the underlying issue here is that for readability it would be better to split into three lines: `begin = f * frame_shift`, `end = f * frame_shift + window_size` and `frame = data[begin:end]`. But not hugely important, I would say.

Reply:
Or actually just `data[begin:begin+window_size]`.